packages/core/src/components/context-menu/_context-menu.scss

```diff
@@ -110,6 +110,10 @@ rightClickMe.oncontextmenu = (e: MouseEvent) => {
 Styleguide components.context-menu.js
 */
 
+.pt-context-menu .pt-popover-target {
+  display: block;
+}
+
 .pt-context-menu-popover-target {
   position: fixed;
 }
```

Comment:
I get that's what those selectors are selecting, but why do we have two sets of class names so confusingly similar?